### PR TITLE
Update roadmap for grammar/clarity

### PR DESCRIPTION
**Pull request by belac9615**
I'm proposing some changes to the roadmap to make the English more concise and easier to read. No other substantial changes.

Signed-off-by: Caleb Angelino <belac9615@users.noreply.github.com>

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [ X ] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [ X ] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -9,16 +9,14 @@ The goal of this document is to give you a hint on what's next. However, since a
 ![](./assets/img/roadmap.png)
 
 ## In tech preview
-
-This categories means the technology is here, but not officially released for production usage at the moment.
+_Technology that is here, but not officially released for production usage._
```
Comment (olivierlambert) [inline docs/roadmap.md:12]: I'd like to get a space after a title, even if it's not visible in the HTML result. Find it more readable when editing the MD file.

File: docs/roadmap.md [hunk 2]
```diff
@@ -9,16 +9,14 @@ The goal of this document is to give you a hint on what's next. However, since a
...
 
 * [IPv6 support in dom0](https://xcp-ng.org/blog/2021/02/09/ipv6-in-xcp-ng/) (Network)
 * [LINSTOR integration](https://xcp-ng.org/blog/2020/11/13/xcp-ng-and-linbit-alliance-part-ii/) (Storage)
 * [RunX](https://xcp-ng.org/blog/2021/10/19/runx-is-available-in-tech-preview/) (Xen)
 
 ## In progress
-
-In there, it means we started to work on it, but it's not usable/visible yet.
+_Things we started to work on, but are not usable/visible yet._
```
Comment (olivierlambert) [inline docs/roadmap.md:19]: Same.

File: docs/roadmap.md [hunk 3]
```diff
@@ -32,8 +30,7 @@ In there, it means we started to work on it, but it's not usable/visible yet.
 * [DPU Support](https://xcp-ng.org/blog/2021/07/12/dpus-and-the-future-of-virtualization/) (storage/platform)
 
 ## Spec/Design/PoC
-
-Those feature are still discussed or designed, not even partly coded.
+_Features that are being discussed or designed, but not even partly coded._
```
Comment (olivierlambert) [inline docs/roadmap.md:33]: Same.

File: docs/roadmap.md [hunk 4]
```diff
@@ -68,7 +65,7 @@ Those feature are still discussed or designed, not even partly coded.
 
 ## Backlog
 
-This is a kind of whish list, without any priorities, where we try to put some ideas.
+This is a kind of wish list, without any priorities, where we try to put some ideas.
```
Comment (olivierlambert) [inline docs/roadmap.md:68]: Should we put this in italic too, like the rest of the content after the title?

**Review by olivierlambert — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! Here is my quick and very minor feedback :)

File: docs/roadmap.md (hunk 4 above)
Comment (belac9615) [reply docs/roadmap.md:68]: Yep, we should. I missed this one. Thanks!

**Comment by belac9615** [comment]
@olivierlambert Thanks for the feedback! I've made the changes that you requested and am submitting an updated patch.

**Review by olivierlambert — APPROVED** [review]

**Comment by olivierlambert** [comment]
Thanks for your contribution, merged and doc rebuilt, result available here: https://xcp-ng.org/docs/roadmap.html

**Pull request merged**